### PR TITLE
Create Tech Fair 2024 Site

### PR DESCRIPTION
**Pull request by micahdbak**

### CODE REVIEW

File: public/tech_fair/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  </style>
+  <body>
+    <pre>
+Welcome to the <b>Tech Fair</b> portal!
+
+<i>You may want to check out...</i>
+
+- <a href="https://sfucsss.org/events/tech_fair/2022/main">Tech Fair 2022</a>
+- <a href="https://sfucsss.org/events/tech_fair/2023">Tech Fair 2023</a>
+- <a href="/tech_fair/2024/index.html">Tech Fair 2024</a>
```
Comment (EarthenSky) [inline public/tech_fair/index.html:43]: these should all be relative links `/tech_fair/2024/` (index.html is implied)

**Review by EarthenSky — COMMENTED** [review]
I was trying to run the page locally, but just executing the html didn't work (this makes sense because it's not being served from a local server). However, `npm run start` and `npm run serve` are also not working, and the logs don't seem to give any info about what's wrong :sob:

EDIT: I just fell back to using my favourite python http server \<3 `python -m http.server 8080 --bind 127.0.0.1 --directory ./`

File: public/tech_fair/index.html (hunk 1 above)
Comment (EarthenSky) [reply public/tech_fair/index.html:43]: Oh, I see the top two are linking to the old site! Can you please add an issue to move them?

File: public/tech_fair/2024/index.html [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+                    >AQ North</a
+                  >
+                  at SFU's Burnaby campus
+                </li>
+                <li>Over 2000 SFU Computing Science Students invited</li>
+                <li>
+                  <b style="color: #84eb7a">October 9th, 2024</b> from 10:00am -
+                  4:00pm
+                </li>
+                <li>Meet your future employees!</li>
```
Comment (EarthenSky) [inline public/tech_fair/2024/index.html:130]: li:before was causing some wonkyness, so I went back to the default bullet, which still looks nice imo.

File: public/tech_fair/index.html (hunk 1 above)
Comment (micahdbak) [reply public/tech_fair/index.html:43]: Hehehehe yup. I'll make an issue! And sure, I'll leave out the index.html

File: public/tech_fair/2024/index.html (hunk 2 above)
Comment (micahdbak) [reply public/tech_fair/2024/index.html:130]: want me to to delete that bullet?

**Review by EarthenSky — APPROVED** [review]
Looks good!

**Pull request merged**